lib/shell/shellscript.js

```diff
@@ -110,13 +117,14 @@ function newScriptError(err, errorFile) {
     return resultErr;
 }
 
-function setVariables(rendition, source, errorFile){
+function setVariables(rendition, source, errorFile, mimeFile){
     // inherit environment variables
     const env = Object.create(process.env || {});
 
     env.source = env.file = stripAnsi(source.path); // escape because user provided
     env.rendition = stripAnsi(rendition.path);
     env.errorfile = errorFile;
+    env.mimefile = mimeFile;
```

Comment:
`mime` itself isn't a commonly used term. Also this file is supposed to include the full content type.

Maybe name this `contenttypefile` or just `typefile` (shorter).

Reply:
I'll rename to `typefile`.